src/sdk/javascript/examples/functions/index.js

```diff
@@ -83,17 +85,24 @@ class IntegratedFunction extends FunctionTalent {
 
     getRules() {
         return new AndRules([
-            new Rule(new OpConstraint(`${this.id}.trigger`, OpConstraint.OPS.GREATER_THAN, 20, DEFAULT_TYPE, VALUE_TYPE_RAW)),
-            new Rule(new ChangeConstraint(`${this.id}.trigger`, DEFAULT_TYPE, VALUE_TYPE_RAW))
+            new Rule(new OpConstraint(`${this.id}.trigger`, OpConstraint.OPS.LESS_THAN_EQUAL, 15, DEFAULT_TYPE, VALUE_TYPE_RAW)),
+            //new Rule(new ChangeConstraint(`${this.id}.trigger`, DEFAULT_TYPE, VALUE_TYPE_RAW))
         ]);
     }
 
     async onEvent(ev, evtctx) {
         const v = TalentInput.getRawValue(ev);
 
-        const sum = await this.call(this.id, 'sum', [ v, v ], ev.subject, ev.returnTopic, 20000);
+        this.logger.info(`${v} + ${v} = ${await this.call(this.id, 'sum', [ v, v ], ev.subject, ev.returnTopic, 20000)}. ${v} x ${v} = ${await this.call('math', 'multiply', [ v, v ], ev.subject, ev.returnTopic)}`, evtctx);
+        this.logger.info(`${v}! = ${await this.call('math','faculty', [ v ], ev.subject, ev.returnTopic, 20000)}`, evtctx);
+
+        try {
+            this.logger.info(`${v}th fibonacci number is ${await this.call('math','fibonacci', [ v ], ev.subject, ev.returnTopic, 5000)}`, evtctx);
+        }
+        catch(err) {
+            console.log(`Could not calculate fibonacci number in the given timeout`);
```

Comment:
Is the comment correct for all error-cases? Could no other error occur instead? (please check to avoid misleading error messages)

Reply:
Yes, this is the only error, which can occur at that specific spot